QUEST_LV_0100.tsv

```diff
@@ -5163,11 +5163,11 @@ QUEST_LV_0100_20150317_005162	Defeat the separated soul of demon
 QUEST_LV_0100_20150317_005163	Kenneth told you to exhaust demon's HP to less than half and use the holiness of barrier to separate the soul and get rid of it. 
 QUEST_LV_0100_20150317_005164	Got rid of enough separated souls in Pumpura hill. Report to Kayetonas.
 QUEST_LV_0100_20150317_005165	Defeat the demon monsters around Kenneth
-QUEST_LV_0100_20150317_005166	Kenneth asked you to kill the monsters nearby while he takes a rest.
-QUEST_LV_0100_20150317_005167	Killed the monsters while Kenneth took rest. Tell Kayetonas about it. 
+QUEST_LV_0100_20150317_005166	Kenneth asked you to defeat the monsters nearby while he takes a rest.
+QUEST_LV_0100_20150317_005167	Defeated the monsters while Kenneth took rest. Tell Kayetonas about it. 
 QUEST_LV_0100_20150317_005168	
-QUEST_LV_0100_20150317_005169	Minatours appeared from the sky like Kayetonas said. Defeat the Minotaurs so that it won't get to the Paladin Master. 
-QUEST_LV_0100_20150317_005170	Defeated Minotaurs with Kayetonas. Talk to Kayetonas.
+QUEST_LV_0100_20150317_005169	Minotaur appeared from the sky like Kayetonas said. Defeat Minotaur so that it won't get to the Paladin Master. 
```

Comment:
That's not exactly the same thing, especially in how this is worded.
Something may be "named" something, such as in this case this being is named "Minotaur".  This refers to what it is, it is a Minotaur.  It's actual name, what it is called as a single being, is different ( but could also be "Minotaur", but that's not likely ).  The structure of the sentence has to change to reflect this.  For example, in some cases we are dealing with a boss monster like "Demon Lord Ghesti".  Her name is Ghesti, and in sentences where she is just called "Ghesti", we change the structure of the sentence to reflect that.  However, she is a "Demon Lord".  If a sentence just referred to her as "Demon Lord", it would have to change to reflect that as well.

You wouldn't say
"Demon Lord appeared from the sky"
You'd say
"The Demon Lord appeared from the sky"
but you -would- say
"Ghesti appeared from the sky"

In this case, I'm sure that this -is- a Minotaur, but we aren't aware of it's name.